bot/app/database/db_objects.py

```diff
@@ -0,0 +1,212 @@
+        curs.close()
+        conn.close()
+
+        return search is not None
+        
+class Artist:
+
+    """An Instance of the Artist relation
+
+    Arguments:
+        * name(str): artist's name
+        * website(str): artist's website
+        * soundcloud(str): artist's soundcloud profile
+        * bandcamp(str): artist's bandcamp profile
+        * bio(str): short descriptive text about the artist. keep it under 150 characters
+        * pic_id(int): ID for corresponding picture. 6 characters long, begins with 1.
+    """
+
+    def __init__(self, name, website, soundcloud, bandcamp, bio, pic_id):
+
+        self.name = name
+        self.website = website
+        self.soundcloud = soundcloud
+        self.bandcamp = bandcamp
+        self.bio = bio
+        self.pic_id = pic_id
+        
+    def insert_artist(self):
+        
+        conn = sqlite3.connect(DB_NAME)
+        curs = conn.cursor()
+
+        try:
+            curs.execute("""
+                insert into Artists values (?,?,?,?,?,?)""",
+                (self.name, self.website, self.soundcloud, self.bandcamp, self.bio, \
+                self.pic_id))
+            self.conn.commit() 
+        except sqlite3.IntegrityError as e:
+            logging.error(e)
+        
+        curs.close()
+        conn.close()
+
+    def is_artist(self):
+
+        conn = sqlite3.connect(DB_NAME)
+        curs = conn.cursor()
+
+        search = curs.execute("""select from Artists where aName=?""", (self.name))
+        
+        curs.close()
+        conn.close()
+
+        return search is not None
+
+class PlaysAt:
+
+    """
+    Instance represents Artist-Event connection (aka playsAt Table)
+    """
+
+    def __init__(self, artist_name, event_name, date):
+        
+        self.artist_name = artist_name
+        self.event_name = event_name
+        self.date = date
+
+    def insert_plays_at(self):
+        
+        conn = sqlite3.connect(DB_NAME)
+        curs = conn.cursor()
+
+        try:
+            curs.execute("""
+                insert into playsAt values (?,?,?)""",
+                (self.artist_name, self.event_name, self.date))
+            self.conn.commit()
+        except sqlite3.IntegrityError as e:
+            logging.error(e)
+        
+        curs.close()
+        conn.close()
+
+class User:
+
+    """Instance of the User relation:
+
+    Arguments:
+        * u_id(str): User-ID, same as the User-ID which Telegram uses
+        * name(str): First name of the user
+        * is_bot(bool): True if the account is a bot
+    """
+
+    def __init__(self, u_id, name, is_bot):
+
+        self.u_id = u_id
+        self.name = name
+        self.is_bot = is_bot
+
+    def user_exists(u_id):
+
+        conn = sqlite3.connect(DB_NAME)
+        curs = conn.cursor()
+
+        curs.execute("select * from Users where uID=?", (self.u_id, ))
+        user = curs.fetchone()
+
+        curs.close()
+        conn.close()
+
+        return true if user is not None 
```

Comment:
`true` is not needed since the expression is evaluated first and the result is returned:

```python
>>> def test(user):
...     return user is not None
...
>>> test("bob")
True
>>> test(None)
False
```

Also `true` is not a keyword in Python, it should be `True` if anything :)

Reply:
fixed